### PR TITLE
ref: Remove `start_transaction` from `OrganizationIntegrationSetupView`

### PR DESCRIPTION
**Pull request by szokeasaurusrex**
This `start_transaction` call is unnecessary and causes undefined behavior. Since the `start_transaction` call occurs within an endpoint, a transaction is already automatically started by the Django integration. The auto-instrumented transaction can be viewed [here](https://sentry.sentry.io/performance/?isDefaultQuery=false&metricSearchSetting=metricsOnly&project=1&query=transaction%3A%22%2Forganizations%2F%7Borganization_slug%7D%2Fintegrations%2F%7Bprovider_id%7D%2Fsetup%2F%22&statsPeriod=24h).

Instead, to preserve the different transaction names for each integration, we now set the transaction name on the auto-instrumented transaction instead of creating a new transaction.

ref GH-63590

### CODE REVIEW

File: src/sentry/web/frontend/organization_integration_setup.py [hunk 1]
```diff
@@ -20,46 +20,34 @@ class OrganizationIntegrationSetupView(ControlSiloOrganizationView):
...
     def handle(self, request: Request, organization, provider_id) -> HttpResponseBase:
-        try:
-            with sentry_sdk.configure_scope() as scope:
-                parent_span_id = scope.span.span_id
-                trace_id = scope.span.trace_id
-        except AttributeError:
-            parent_span_id = None
-            trace_id = None
+        with sentry_sdk.configure_scope() as scope:
+            scope.transaction.op = "integration.setup"
```
Comment (szokeasaurusrex) [inline src/sentry/web/frontend/organization_integration_setup.py:24]: is it important to be setting the `op` here as well?

File: src/sentry/web/frontend/organization_integration_setup.py [hunk 2]
```diff
@@ -20,46 +20,34 @@ class OrganizationIntegrationSetupView(ControlSiloOrganizationView):
...
+            scope.transaction.name = f"integration.{provider_id}"
```
Comment (sl0thentr0py) [inline src/sentry/web/frontend/organization_integration_setup.py:25]: please use this method and set the source route/view (both are fine). The source is important to tell ingest that the transaction name is 'low cardinality'.

https://github.com/getsentry/sentry-python/blob/e864eab559c2b37b44bdf6f353cbdb25c8f885ce/sentry_sdk/scope.py#L499-L501

File: src/sentry/web/frontend/organization_integration_setup.py (hunk 1 above)
Comment (sl0thentr0py) [reply src/sentry/web/frontend/organization_integration_setup.py:24]: no, the op will just be the normal django http request stuff from the integration.

**Review by sl0thentr0py — CHANGES_REQUESTED** [review]

File: src/sentry/web/frontend/organization_integration_setup.py [hunk 3]
```diff
@@ -20,46 +20,35 @@ class OrganizationIntegrationSetupView(ControlSiloOrganizationView):
...
-            name=f"integration.{provider_id}",
-            parent_span_id=parent_span_id,
-            trace_id=trace_id,
-            sampled=True,
-        ):
-            pipeline = IntegrationPipeline(
-                request=request, organization=organization, provider_key=provider_id
+        with sentry_sdk.configure_scope() as scope:
+            scope.set_transaction_name(
+                f"integration.{provider_id}", source="OrganizationIntegrationSetupView"
```
[the remark this answers is not in the document]
Comment (sl0thentr0py) [reply src/sentry/web/frontend/organization_integration_setup.py:25]: source has to be one of those
https://github.com/getsentry/sentry-python/blob/e864eab559c2b37b44bdf6f353cbdb25c8f885ce/sentry_sdk/tracing.py#L42-L47

**Review by sl0thentr0py — CHANGES_REQUESTED** [review]

File: src/sentry/web/frontend/organization_integration_setup.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (szokeasaurusrex) [reply src/sentry/web/frontend/organization_integration_setup.py:25]: ahh okay, I thought you meant to set the `source` to the route or view

**Review by sl0thentr0py — APPROVED** [review]

**Pull request merged**